src/component/page/profile/index.tsx

```diff
@@ -27,6 +27,8 @@ function Profile(props: ProfileProps) {
 
   if (userLoading) {
     return <span>Loading User</span>;
+  } else if (user && hasInvite) {
```

Comment:
The redirect URI from Auth0 is `/profile` which is this component. If the user already has a profile (actually a "User" in the API), we redirect to root and miss the fact that the user was invited. With this check, we can go to the join venture page if the invite is still present in localstorage.